### ISSUE TITLE
Fix lil_interpret function

### ISSUE THREAD

**Issue by asundaresan** [asundaresan/SelfExplain#3]
The lil_interpret function has the following definition: 

`def lil_interpret(logits, list_of_interpret_dict, dev_samples, current_idx):`

where `current_idx` is the index into `dev_samples` so that it is aligned with `logits`. It is much simpler to do the alignment outside the function so that the function definition is as follows. 

`def lil_interpret(logits, list_of_interpret_dict, dev_samples):`

The alignment can be done outside the function:

`dev_samples = dev_samples_all[current_idx:current_idx+batch_size]`
